src/main/java/com/epam/ta/reportportal/core/events/handler/TestItemFinishedEventHandler.java

```diff
@@ -53,11 +55,11 @@ public void onApplicationEvent(ItemFinishedEvent itemFinishedEvent) {
 
 		AnalyzerConfig analyzerConfig = AnalyzerUtils.getAnalyzerConfig(project);
 
-		logIndexer.indexItemsLogs(
+		CompletableFuture.supplyAsync(() -> logIndexer.indexItemsLogs(
```

Comment:
maybe we need the whole event async here that is executed after main transaction is commited

Reply:
fixed